src/field_theory/minpoly/field.lean

```diff
@@ -84,6 +84,16 @@ lemma dvd_map_of_is_scalar_tower (A K : Type*) {R : Type*} [comm_ring A] [field
   minpoly K x ∣ (minpoly A x).map (algebra_map A K) :=
 by { refine minpoly.dvd K x _, rw [aeval_map_algebra_map, minpoly.aeval] }
 
+lemma dvd_map_of_is_scalar_tower' {R S : Type*} (K L : Type*) [comm_ring R]
```

Comment:
`R` should be explicit here (this could also solve your problem with `all_goals`, but I am not sure).

Reply:
And also `s`.